app/views/events/show.html.erb

```diff
@@ -36,3 +36,36 @@
 
 <%= link_to 'Edit', edit_event_path(@event) %> |
 <%= link_to 'Back', events_path %>
+
+<%= form_with model: [@event, @comment] do |f| %>
+  <%= f.text_field :body %>
+  <%= f.submit "comment", disable_with: 'commenting...' %>
+<% end %>
+
+
+<div>
+  <% @comments.each do |comment| %>
+    [<%= comment.created_at.strftime("%I:%M %p; %d-%m-%y")%>] 
+    <%= comment.user.name%> : 
+    <%= comment.body %>
+      <% if @current_user && @current_user == comment.user %>
+        <%= button_to 'Destroy', event_comment_path(@event, comment), method: :delete, class: 'btn btn-danger'%>
+      <% end %>
+
+      <%= form_with model: [comment, @answer] do |d| %>
```

Comment:
Please change `d` for something more understandable

Reply:
I have finished making the change